packages/contracts/README.md

```diff
@@ -70,6 +70,79 @@ Then, copy the deployment address and paste it in to replace `DEPLOYED_CONTRACT_
 npx hardhat verify --network goerli DEPLOYED_CONTRACT_ADDRESS "Hello, Hardhat!"
 ```
 
+## Testing
+
+The `@aragon/osx-versions` package allows you to work with previous versions of the contracts. The solidity source files of these versions are available in the `packages/contracts-versions/versions` directory after the package was built. For a details please refer to the [packages/contracts-versions/README.md](https://github.com/aragon/osx/blob/develop/packages/contracts-versions/README.md).
+
+If you want to import and test a contract from a previous version, you must first import the contract into `contracts/src/tests/osx-versions/Migration.sol`. This step is necessary for the contract to be compiled.
+
+```solidity
+// contracts/src/tests/osx-versions/Migration.sol
+
+import '../../contracts-versions/{version-name}/{path_to_contract}.sol';
```

Comment:
I would also mention that the contract should be renamed with its own version, so that there are not 2 contract artifacts called `DAO`

Reply:
No, we do not rename the contracts,
and there is will be more than 1 contract with the same name
but since we import from a different package we have their artifact in there respective folder
for example:
`artifacts/@aragon/osx-versions/versions/{versions-name}/dao/DAO.sol/DAO.json`

Reply:
I do think we rename them upon import:
<img width="522" alt="image" src="https://github.com/aragon/osx/assets/4456749/c09cf01a-4cfa-4daa-bc52-36164af647b8">